packages/zeebe-element-templates-json-schema/test/spec/validationSpec.js

```diff
@@ -49,7 +49,12 @@ function createTest(name, file, it) {
     } = validateTemplate(template);
 
     // then
-    expect(errors).to.eql(expectedErrors);
+    expect(errors, `\n
+    expected:
+      ${expectedErrors?.length ? JSON.stringify(expectedErrors, null, 1).replace(/"([^"]+)":/g, '$1:') : 'no errors'} 
+    but got actual:
+      ${JSON.stringify(errors, null, 1).replace(/"([^"]+)":/g, '$1:')}\n`)
+      .to.eql(expectedErrors);
```

Comment:
We could reuse jsonequal matcher: https://github.com/bpmn-io/bpmn-js/tree/13f36155d65b69355fa17398798be6dc4d307e6f/test/matchers

Reply:
So this could be done as a separate change.

Reply:
So, should I exclude this here?

Reply:
Please do but feel free to create a separate PR. Notice that this change is implemented only in one of the schemas. Ideally, this should be a shared helper.